lib/ferrum/browser.rb

```diff
@@ -77,6 +77,10 @@ def extensions
       end
     end
 
+    def evaluate_on_new_document
```

Comment:
I think we should reuse what we have for extensions now and modify `@extensions` array calling this method